pytripgui/leftmenu.py

```diff
@@ -295,7 +297,7 @@ def plan_export_cube(self, evt):
     def plan_let_add(self, msg):
         plan = msg.data["plan"]
         let = msg.data["let"]
-        name = msg.data["name"]
+        # name = msg.data["name"]
```

Comment:
I removed it, but in general sometimes I prefer to leave such comments in, when I suspect they may become useful later. I might else forget that the 'name' is available as key.